build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     // jakarta.json-api is not exposed on any public members, so does not need
     // to be exposed to use the library
     implementation 'jakarta.json:jakarta.json-api:2.1.3'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.16.0'
```

Comment:
Is there a jakarta lib that would work for this rather than jackson? (and is that a reasonable idea)
Possibly this: https://projects.eclipse.org/projects/ee4j.jsonb


The general goal of this library was to avoid taking a dependency on any specific JSON api, and I think at the time the jakarta ones were the closest to being a general spec. It's possible that jackson is more widely used, but it's been a long while since I've written any Java or looked at this library. You probably have more up to date context about these things than I do currently (or could perhaps dig into this to understand whether this is the right approach).